promql/engine.go

```diff
@@ -529,6 +531,7 @@ func (ng *Engine) populateSeries(ctx context.Context, q storage.Queryable, s *Ev
 				maxOffset = n.Offset + LookbackDelta + subqOffset
 			}
 		case *MatrixSelector:
+			aggregationWindow = n.Range.Nanoseconds() / 1000000
```

Comment:
`1000000` -> `time.Millisecond`?

Reply:
time.Millisecond is Duration which means ill have to cast it again..